portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -39,15 +40,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             result += "\"status\": \"true\", ";
             result += "\"url\": \""+ logoutURL + "\"";
             result += "}";
-
-            response.setContentType("application/json;"); 
+            
             response.getWriter().println(result);
             return;
         }
         String loginURL = userService.createLoginURL(urlToRedirectTo);
 
         String result = "{";
-        result += "\"status\": \"true\", ";
+        result += "\"status\": \"false\", ";
```

Comment:
This is also a bug we missed. Same as above, please explain this in the PR description.

Reply:
The PR description is updated! :)